src/include/distributed/utils/attribute.h

```diff
@@ -0,0 +1,66 @@
+/*-------------------------------------------------------------------------
+ *
+ * attribute.h
+ *	  Routines related to the multi tenant monitor.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#ifndef CITUS_ATTRIBUTE_H
+#define CITUS_ATTRIBUTE_H
+
+#include "executor/execdesc.h"
+#include "executor/executor.h"
+#include "storage/lwlock.h"
+
+typedef struct TenantStats
+{
+	char tenantAttribute[100];
+
+	int colocationGroupId;
+
+	int selectCount;
+	double totalSelectTime;
+	int selectsInLastPeriod;
+	int selectsInThisPeriod;
+
+	int insertCount;
```

Comment:
I probably missed something. Why are we specifically tracking inserts and not other writes?

Reply:
I think I should add them. Should it be `INSERT/UPDATE/DELETE` for writes and `SELECT` for reads

Reply:
Updated my PR and added `UPDATE` and `DELETE` queries